src/modules/chefs/api.ts

```diff
@@ -1,24 +1,27 @@
 import axios from "axios";
 
+const BASE_URL = process.env.REACT_APP_BASE_URL;
+const chefsBaseUrl = `${BASE_URL}/chefs`;
+
 export async function getAllChefsFromAPI(selectedCategoryItem : any){
     const result = await axios.get(
-        `http://localhost:5000/chefs?category=${selectedCategoryItem}`
+        `${chefsBaseUrl}?category=${selectedCategoryItem}`
     );
     
     return result;
 }
 
 export async function deleteChefFromAPI(id: any){
     const result = await axios.delete(
-        `http://localhost:5000/chefs/deleteChef/${id}`
+        `${chefsBaseUrl}/deleteChef/${id}`
```

Comment:
there is no need to write deletechef, addchef and updatechef since they use different methods (GET,POST,PUT,DELETE, etc..) they can all end with chefs and the difference could be the method, also in get all and get they can be differentiated by parameter